tests/components/flux/test_switch.py

```diff
@@ -723,10 +723,8 @@ def event_date(hass, event, now=None):
     assert call.data[light.ATTR_XY_COLOR] == [0.439, 0.37]
 
 
-@pytest.mark.parametrize("x", [0, 1])
```

Comment:
Indeed, a lonely parametrize value that wasn't doing anything.

I tried to find what the intention of this parameter was, but couldn't see it, including in the original commit.